src/utils/map-state.js

```diff
@@ -53,6 +53,15 @@ export default class MapState {
     */
     altitude,
 
+    /** Viewport transitions */
+    transitionDuration,
```

Comment:
`MapState` does not consume these props, though it is used to normalize viewport props before sending them to callbacks.